meta/christianbarkowsky/contao-tiny-compress-images/de.yml

```diff
@@ -1,11 +1,17 @@
 de:
-  title: JPG- und PNG-Komprimierung
+  title: JPG, PNG und WebP Komprimierung
```

Comment:
The linter does report `WebP` as a spelling error. Could you add `WebP` to the `default.txt` allow list?

![image](https://user-images.githubusercontent.com/1218766/147687236-c5cb82a5-4afc-42af-bdce-5cba976d6d79.png)